app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/activity/InfoActivityLayout.kt

```diff
@@ -229,10 +230,47 @@ fun InfoActivityLayout(
     }
 }
 
+@Composable
+fun ContributorsList(
+    modifier: Modifier = Modifier
+) {
+    val contributorsText = remember {
+        contributors.joinToString(separator = "\n") {
+            "${it.name} (@${it.username})".asListItem()
+        }
+    }
+    Column(modifier = modifier.fillMaxWidth()) {
+        Text(
+            modifier = Modifier.padding(top = 16.dp),
+            text = stringResource(id = R.string.contributors_list_title),
+            style = MaterialTheme.typography.headlineMedium,
+            textAlign = TextAlign.Center
+        )
+        Text(
+            modifier = Modifier.fillMaxWidth(),
```

Comment:
Could you add `.padding(top = 16.dp, bottom = 8.dp)` here?
The top padding will make it consistent with the other headers and the bottom padding will better separate the sentence from the following list.

Reply:
Done